src/safeposix/syscalls/net_calls.rs

```diff
@@ -1318,11 +1322,6 @@ impl Cage {
                     return syscall_error(Errno::EINVAL, "epoll wait", "max events argument is not a positive number");
                 }
 
-                //now we know that all of the arguments are valid...
-                let mut reads = interface::RustHashSet::<i32>::new();
-                let mut writes = interface::RustHashSet::<i32>::new();
-                let mut errors = interface::RustHashSet::<i32>::new();
```

Comment:
I think that these are in the original system call from Repy... are they unnecessary when using Rust?

Reply:
Yeah they're not actually used in poll. I think they were unused in repy as well, but probably dated back from when poll was handled separately from select in repy.